shopping-cart.html

```diff
@@ -104,6 +104,98 @@ <h2 class="shopping-cart__left-text">Mi carrito</h2>
         </div>
       </div>
     </section>
+    <section class="purchase-summary">
+      <div class="purchase-summary-top">
+        <div class="purchase-summary__background">
+          <div class="purchase-summary__send">
```

Comment:
Creo que ese div parent esta de mas. Apuesto que puedes hacer esto sin usarlo y pasando el estilo del color de fondo a `purchase-summary__send`
```suggestion
        <div class="purchase-summary__send">
```